rules/certificates.bzl

```diff
@@ -49,6 +50,10 @@ def _certificate_codegen_impl(ctx):
         )
 
     return [
+        CcInfo(compilation_context = cc_common.create_compilation_context(
```

Comment:
Currently, the code is compiled by a `cc_library` call in the `certificate_codegen` macro. If I understand correctly, this introduces an alternative way to compile the code?

Reply:
oops, sorry, this is not needed since I fixed path --> basenem. (I removed it.)